ansible/playbooks/etcd-setup.yml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
> 
> 
> I usually make the main playbook idem-potent, doing everything which makes sense at least. So you can run it again and again (= deploy + configure). Restarts are done with notifiers (compare https://github.com/sttts/compute-platform/blob/master/roles/mesos/tasks/master.yml#L8).

One can not run notifier without running a task that actually notifies a notifier. So this way is no way. Hard to say how many use cases there are which require only restart. I can imagine someone installing/updating etcd or docker manually (followed by manual configuration) without actually running install/configure tasks. So keeping the restart step separate out of notifiers make sense.

> 
> Those steps which do not fit in this scheme (or are dangerous to do automatically), I split them out in special playbooks.
> 

There is adhoc directory exactly for that (or at least that is my impression)